src/ai/backend/agent/docker/utils.py

```diff
@@ -10,9 +10,10 @@
 from aiodocker.exceptions import DockerError
 
 from ai.backend.common.logging import BraceStyleAdapter
+from ai.backend.common.utils import update_nested_dict
```

Comment:
I used absolute path `from ai.backend.common.utils` instead of the relative path `from ...common.utils `